doc/Changelog.md

```diff
@@ -1,6 +1,11 @@
 Changelog {#Changelog}
 =========
 
+## git master {master)
+
+* Add brion::Target::parse() to resolve a given target name
```

Comment:
Never been done here :P

Will do, but in general annoying: git push, open pr, edit changelog, git commit amend, git force push, retest this please...